src/commands/argument.js

```diff
@@ -56,6 +56,12 @@ class Argument {
 		 */
 		this.prompt = info.prompt;
 
+		/**
+		 * Show cancellation message for argument
+		 * @type {boolean}
+		 */
+		this.showCancel = typeof info.showCancel !== 'undefined' ? info.showCancel : true;
```

Comment:
To be consistent with how it works in commands/base.js, you should use
`this.showCancel = 'showCancel' in info ? info.showCancel : true;`

Reply:
Actually, you're consistent with how it works within arguments.js, it's just that arguments.js and base.js are inconsistent between themselves, this should be corrected in a separate PR, nvm this.